src/components/app-configure/access/GenerateKeySuccess.vue

```diff
@@ -0,0 +1,116 @@
+<script lang="ts" setup>
+import CopyIcon from '@/assets/iconography/copy.svg'
+import VButton from '@/components/lib/VButton/VButton.vue'
+import VCard from '@/components/lib/VCard/VCard.vue'
+import VOverlay from '@/components/lib/VOverlay/VOverlay.vue'
+import VStack from '@/components/lib/VStack/VStack.vue'
+import { useToast } from '@/components/lib/VToast'
+import copyToClipboard from '@/utils/copyToClipboard'
+
+const emit = defineEmits(['proceed'])
+
+const props = defineProps({
+  delegatePrivateKey: {
+    type: String,
+    required: true,
+  },
+})
+
+const toast = useToast()
+
+async function copyAddress() {
+  try {
+    await copyToClipboard(props.delegatePrivateKey)
+    toast.success('Address copied')
+  } catch (e) {
+    console.error(e)
+    toast.error('Failed to copy. Try again or contact support')
+  }
+}
+</script>
+
+<template>
+  <VOverlay>
+    <div class="center height-100 width-100">
+      <VCard
+        popup="true"
+        class="flex column"
+        style="max-width: 600px; padding: 30px"
```

Comment:
Move this to `<style>`

Reply:
You mean to `.css`

Reply:
No, there is `<style scoped>` at the bottom, I guess we can move internal styles there

Reply:
https://github.com/arcana-network/developer-dashboard/pull/156/commits/abd5da92c44f00b43b56e3b16c0b885131ed0a11